specification/ServiceBusManagement.json

```diff
@@ -0,0 +1,413 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-04",
+    "title": "ServiceBusManagementClient",
+    "description": "The Azure ServiceBus Management API."
+  },
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/xml"
+  ],
+  "consumes": [
+    "application/xml"
+  ],
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{namespaceName}.servicebus.windows.net",
+    "parameters": [
+      {
+        "name": "namespaceName",
+        "description": "The name of the ServiceBus namespace to send requests to.",
+        "required": true,
+        "type": "string",
+        "in": "path",
+        "x-ms-skip-url-encoding": true
+      }
+    ]
+  },
+  "paths": {
+    "/{queuePath}/": {
+      "get": {
+        "tags": [
+          "Queues"
+        ],
+        "operationId": "Queue_Get",
+        "description": "Get the details about the queue found at the provided queuePath.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/QueuePathParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/EnrichFalseParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Get Queue operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/GetQueueResponse"
```

Comment:
Let's name this `Queue`. This will ensure that the generated interface is called Queue.